app/routes/missions.fermi/route.mdx

```diff
@@ -1,3 +1,4 @@
+import { WithFeature } from '~/root'
```

Comment:
Re-arrange the line as:
---
handle:
  breadcrumb: Fermi
---

import { WithFeature } from '~/root'

import logo from './logo.svg'